src/reference/asciidoc/stream.adoc

```diff
@@ -0,0 +1,138 @@
+[[stream-support]]
+=== Using the RabbitMQ Stream Plugin
+
+Version 2.4 introduces initial support for the https://github.com/rabbitmq/rabbitmq-stream-java-client[RabbitMQ Stream Plugin Java Client] for the https://rabbitmq.com/stream.html[RabbitMQ Stream Plugin].
```

Comment:
Why "initial"? 
Is there anything else we could add on top of existing API or something else?

Reply:
The current implementation (client/plugin) doesn't support concurrency - it's coming in a future release.